README.md

```diff
@@ -67,8 +67,8 @@ variables provide additional configuration:
 Example for running with REST API:
 
 ```
-    $ export GA4GH_VRS_DATAPROXY_URI=https://services.genomicmedlab.org/seqrepo
-    $ export ANYVAR_STORAGE_URI="redis:///15"
+    $ export GA4GH_VRS_DATAPROXY_URI=seqrepo+https://services.genomicmedlab.org/seqrepo
+    $ export ANYVAR_STORAGE_URI=postgres://postgres:postgres@localhost/anyvar_db
```

Comment:
can we keep the documentation for how to use redis?

Reply:
@andreasprlic updated in latest commit